packages/lodestar/src/api/impl/validator/index.ts

```diff
@@ -227,20 +226,16 @@ export function getValidatorApi({
       // the first `MAXIMUM_GOSSIP_CLOCK_DISPARITY` duration of the epoch `tolerantCurrentEpoch`
       // will equal `currentEpoch + 1`
 
-      const duties: routes.validator.AttesterDuty[] = [];
-      for (const validatorIndex of validatorIndices) {
-        const validator = state.validators[validatorIndex];
+      const validators = state.validators.persistent;
+      const validatorData: BLSPubkey[] = [];
```

Comment:
Should this be a Map? This is going to be a hugely sparse array. Not sure if it's dangerous memory-wise for Node.

Reply:
I'm not sure either.  @wemeetagain thoughts?

Reply:
I don't think its dangerous, but there's no harm in using a Map to make this clearer